internal/app/machined/pkg/controllers/hardware/system.go

```diff
@@ -62,48 +64,51 @@ func (ctrl *SystemInfoController) Run(ctx context.Context, r controller.Runtime,
 	case <-r.EventCh():
 	}
 
-	// controller runs only once
-	if ctrl.SMBIOS == nil {
-		s, err := pkgSMBIOS.GetSMBIOSInfo()
-		if err != nil {
-			return err
-		}
-
-		ctrl.SMBIOS = s
-	}
-
-	const systemInfoID = "systeminformation"
-	if err := r.Modify(ctx, hardware.NewSystemInformation(systemInfoID), func(res resource.Resource) error {
-		hwadapter.SystemInformation(res.(*hardware.SystemInformation)).Update(&ctrl.SMBIOS.SystemInformation)
+	if ctrl.V1Alpha1Mode != runtimetalos.ModeContainer {
```

Comment:
we can make it dead simple - if `ctrl.V1Alpha1Mode == runtimetalos.ModeContainer { return nil }` in the beginning of the method. Controller will stop and it won't be restarted.

Reply:
fixed :+1: